autosave/mixins.py

```diff
@@ -168,7 +169,7 @@ def autosave_media(self, obj=None, get_params=''):
 
         return forms.Media(js=(
             reverse('admin:%s_%s_autosave_js' % info, args=[pk]) + get_params,
-            "%sautosave/js/autosave.js?v=2" % settings.STATIC_URL,
+            static('autosave/js/autosave.js'),
```

Comment:
@whatisjasongoldstein not really; there's a standard one if you use FileStorage, and a mixin if not. I just have:

```python
from storages.backends.s3boto import S3BotoStorage
from django.contrib.staticfiles.storage import CachedFilesMixin


class S3HashedFilesStorage(CachedFilesMixin, S3BotoStorage):
    """
    Extends S3BotoStorage to also save hashed copies (i.e.
    with filenames containing the file's MD5 hash) of the
    files it saves.
    """
    pass
```